### PR TITLE
fix(Docs): Add babel-polyfill so that the Docs work in IE11

### PR DESCRIPTION
**Pull request by mihai-dinculescu**

### CODE REVIEW

**Comment by fracmak** [comment]
Can you throw in 

useBuiltIns: 'entry'

Into the .babelrc file for the babel-preset-env plugin?

File: package.json [hunk 1]
```diff
@@ -90,6 +90,7 @@
     "babel-plugin-lodash": "^3.2.11",
     "babel-plugin-transform-react-handled-props": "^1.0.0",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.13",
+    "babel-polyfill": "^6.26.0",
```
Comment (layershifter) [inline package.json]: We use Babel 7

**Review by mihai-dinculescu — COMMENTED** [review]
@fracmak, like this?

**Comment by fracmak** [comment]
yup

File: build/preset-env.js [hunk 2]
```diff
@@ -26,6 +26,7 @@ const browsers = [
 
 const envOptions = {
   targets: { browsers },
+  useBuiltIns: 'entry',
```
Comment (layershifter) [inline build/preset-env.js]: This change will affect also `dist` build that will be published to NPM. We also using [transform runtime](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/.babelrc#L16).

I am not sure that proposed changes are safe.

Comment (mihai-dinculescu) [reply build/preset-env.js]: I'm not confident that this setting helps the docs. Albeit crude, my tests showed no difference in network traffic. I'll take it out for now.

**Comment by levithomason** [comment]
This is an easy merge, however, what feature are we using that is not IE 11 compliant?  This is a target browser.

**Comment by mihai-dinculescu** [comment]
`Promise`

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.81.2`.

**Pull request merged**